### DISCUSSION TITLE
[Enhancement] [table-v2] 表格自定义列选择器工具栏

### DISCUSSION THREAD

**Discussion by softboy99** [element-plus/element-plus#18718] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table-v2`

## Description

自定义表格显示的列
可参考vxetable的工具栏

### Additional comments

_(empty)_

![image](https://github.com/user-attachments/assets/9633a101-0428-4db0-b6bd-e3c3009367b9)

**Comment by elarbor** [comment]
表格就该用一个组件，就叫`<el-table>` 。现在是两个[`<el-table>` ](https://element-plus.org/zh-CN/component/table.html)和[`<el-table-v2>`](https://element-plus.org/zh-CN/component/table-v2.html) 。

用type区分更好一点：
`<el-table type="default">` 就是原来的`<el-table>`组件
`<el-table type="virtualized">`  就是原来的`<el-table-v2>` 组件
`<el-table type="第三种">`  就是以后的`<第三种>` 组件。    ......
